app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java

```diff
@@ -182,8 +183,15 @@ public void handleSpeechResult(String spokenText) {
      */
     public void removeDescription(final UploadMediaDetail uploadMediaDetail, final int position) {
         selectedLanguages.remove(position);
-        final int ListPosition =
-            (int) selectedLanguages.keySet().stream().filter(e -> e < position).count();
+        final int ListPosition;
```

Comment:
No need to reassign ListPosition - just make it not final and use it directly to count.